Tinyxml2Serializer.h

```diff
@@ -90,7 +90,7 @@ namespace qualisys_cpp_sdk {
         std::uint32_t mnMinorVersion;
         void AddXMLElementBool(tinyxml2::XMLElement& parent, const char* tTag, const bool* pbValue, tinyxml2::XMLDocument& oXML, const char* tTrue = "True", const char* tFalse = "False");
         void AddXMLElementBool(tinyxml2::XMLElement& parent, const char* tTag, const bool pbValue, tinyxml2::XMLDocument& oXML, const char* tTrue = "True", const char* tFalse = "False");
-        void AddXMLElementInt(tinyxml2::XMLDocument* oXML, const char* tTag, const int* pnValue);
+        void AddXMLElementInt(tinyxml2::XMLElement& parent, const char* tTag, const int* pnValue, tinyxml2::XMLDocument& oXML);
```

Comment:
Suggestion: Consider renaming the formal parameter oXML into pXmlDocument

Reply:
Shall we do one larger clean-up of the Feature Branch before it goes into master?
- Remove comments if we deem them to provide no value
- Rename variables to be consistent
etc.